build/helper/documentation_snippets.py

```diff
@@ -105,6 +114,29 @@
 '''
 
 
+def close_function_def_for_doc(functions, config):
+    # This is very specific to session based APIs. We look for a 'close' function and if we find it,
```

Comment:
I think this is missing context and is now a bit outdated. Suggestion:


> This is very specific to session based APIs. We look for a 'close' function in the metadata which is to become a code-generated "private" method of Session for which we provide a public wrapper.
Copy its documentation so we apply it to the said public wrapper.

Reply:
Changed